15.0/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN set -x; \
         && /install/setup-pip.sh \
         && /install/postgres.sh \
         && /install/kwkhtml_client.sh \
+        && /install/kwkhtml_client_force_python3.sh \
```

Comment:
according to this  https://github.com/camptocamp/docker-odoo-project/commit/75992d08b7bc64debf52192d78ccc001139aef76#diff-379d1b6436dbc1f0267c62cfb7463cd321ca5b543c3cd41c6ec56a6008455340L39
We should set an `KWKHTMLTOPDF_SERVER_URL` which I don't find (I probably missed it though).
Have you tried this?